src/Microsoft.AspNetCore.SignalR.Core/DefaultHubLifetimeManager.cs

```diff
@@ -129,6 +129,25 @@ public override Task InvokeGroupAsync(string groupName, string methodName, objec
             return Task.CompletedTask;
         }
 
+        public override Task InvokeGroupExceptAsync(string groupName, string methodName, object[] args, IReadOnlyList<string> excludedIds)
+        {
+            if (groupName == null)
+            {
+                throw new ArgumentNullException(nameof(groupName));
+            }
+
+            var group = _groups[groupName];
+            if (group != null)
+            {
+                var message = CreateInvocationMessage(methodName, args);
+                var tasks = group.Values.Where(connection => !excludedIds.Contains(connection.ConnectionId))
+                    .Select(c => c.WriteAsync(message));
+                return Task.WhenAll(tasks);
+            }
```

Comment:
Is this a reasonable place to add logging or should we first finalize our logging strategy? @BrennanConroy @anurse